### PR TITLE
add startOz

### PR DESCRIPTION
**Pull request by Tatenda-D-Mawoneke**

### CODE REVIEW

File: Warmup-1/startOz/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export function startOz(givenInput: string): string {
```
Comment (JustinGuedes) [inline Warmup-1/startOz/src/index.ts:1]: Please simplify this function

File: Warmup-1/startOz/test/index.test.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+// Given a string, return a string made of the first 2 chars (if present), however include first char only if it is 'o' and include the second only if it is 'z', so "ozymandias" yields "oz".
+
+import { assert } from "chai";
+import { startOz } from "../src";
+
+// startOz("ozymandias") → "oz"
+// startOz("bzoo") → "z"
+// startOz("oxx") → "o"
+
+describe("startOz", () => {
```
Comment (JustinGuedes) [inline Warmup-1/startOz/test/index.test.ts:10]: Please add a test for when there is no `o` or `z` in the string

**Review by JustinGuedes — CHANGES_REQUESTED** [review]

File: Warmup-1/startOz/src/index.ts (hunk 1 above)
Comment (Tatenda-D-Mawoneke) [reply Warmup-1/startOz/src/index.ts:1]: Sure.

File: Warmup-1/startOz/test/index.test.ts (hunk 2 above)
Comment (Tatenda-D-Mawoneke) [reply Warmup-1/startOz/test/index.test.ts:10]: Done.

File: Warmup-1/startOz/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  // const myStringArray = [...givenInput];
```
Comment (JustinGuedes) [inline Warmup-1/startOz/src/index.ts:2]: Please remove commented out code

**Review by JustinGuedes — CHANGES_REQUESTED** [review]

File: Warmup-1/startOz/src/index.ts (hunk 3 above)
Comment (Tatenda-D-Mawoneke) [reply Warmup-1/startOz/src/index.ts:2]: Done

**Review by JustinGuedes — APPROVED** [review]

**Pull request merged**